specs/hotel_spec.rb

```diff
@@ -0,0 +1,147 @@
+require_relative 'spec_helper'
+
+describe "Hotel class" do
+
+  describe "Instantiation" do
+
+    it "Can be instantiated" do
+      Hotel::Hotel.new(20, 200).must_be_instance_of Hotel::Hotel
+    end
+
+    it "Can create an array of rooms" do
+      Hotel::Hotel.new(20, 200).rooms.length.must_equal 20
+    end
+
+    it "Can call the array of rooms" do
+      Hotel::Hotel.new(20,200).rooms.must_equal [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18, 19, 20]
+    end
+
+    it "Can call on price of rooms" do
+      Hotel::Hotel.new(20, 200).price.must_equal 200
+    end
+
+    it "can call on collection of reservations array - will be empty" do
+      Hotel::Hotel.new(20, 200).reservation_collection.must_be_instance_of Array
+      Hotel::Hotel.new(20, 200).block_reservation_collection.must_be_instance_of Array
+
```

Comment:
Two observations about this test:
- You probably don't want to instantiate two different hotels
- You don't actually check that the array is empty

You might rewrite it as follows:

```ruby
it "can call on collection of reservations array - will be empty" do
  hotel = Hotel::Hotel.new(20, 200)

  hotel.reservation_collection.must_be_instance_of Array
  hotel.reservation_collection.length.must_equal 0

  hotel.block_reservation_collection.must_be_instance_of Array
  hotel.block_reservation_collection.length.must_equal 0
end
```